packages/ubuntu_desktop_installer/lib/installer.dart

```diff
@@ -256,8 +264,17 @@ class _UbuntuDesktopInstallerWizard extends StatelessWidget {
     return Wizard(
       initialRoute: initialRoute ?? Routes.initial,
       routes: <String, WizardRoute>{
-        Routes.welcome: const WizardRoute(
+        Routes.welcome: WizardRoute(
           builder: WelcomePage.create,
+          onNext: (_) {
+            if (tryOrInstall == true) {
+              return Routes.tryOrInstall;
```

Comment:
these lines are covered by the integration test even though we don't collect coverage info for them because by running the entire app it'd be too easy to cover 100% :P